src/api/entities/confidential/ConfidentialAccount/types.ts

```diff
@@ -34,3 +36,15 @@ export type ConfidentialAssetHistoryEntry = {
   amount: string;
   createdAt: EventIdentifier | null;
 };
+
+export interface ApplyIncomingConfidentialAssetBalancesParams {
+  /**
+   * Confidential Account (or the public key of the ElGamal key pair) to which any incoming balance is to be applied
+   */
+  confidentialAccount: string | ConfidentialAccount;
+
+  /**
+   * The maximum number of incoming balances to apply
+   */
+  maxUpdates: BigNumber;
```

Comment:
What about making this optional and defaulting to all? 

I am not sure on the use case here. I figure I would want to apply all incoming balances most of the time though.